File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.cs
public class Solution {
    public int MaxFrequencyElements(int[] nums) {
        Dictionary<int, int> md = new Dictionary<int, int>();
        foreach(int num in nums)
        {
            if(md.ContainsKey(num))
            {
                md[num]+=1;
            }
            else
            {
                md[num]=1;
            }
        }
        int x = 0;
        int mx = md.Values.Max();
        foreach(KeyValuePair<int, int> m in md)
        {
            if(m.Value==mx)
            {
                x+=m.Value;
            }
        }
        return x;
    }
}

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.py
class Solution(object):
    def maxFrequencyElements(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n={}
        for num in nums:
            if num in n:
                n[num]+=1
            else:
                n[num]=1
        x=0
        v=max(n.values())
        for i, j in n.items():
            if j==v:
                x+=v
        return x
